tests/expireat_test.go

```diff
@@ -0,0 +1,55 @@
+package tests
+
+import (
+	"gotest.tools/v3/assert"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func TestEvalEXPIREAT(t *testing.T) {
+	conn := getLocalConnection()
+
+	expireInSeconds := int64(10) // Set expiration time to 10 seconds later
+
+	testCases := []struct {
+		name     string
+		commands []string
+		expected []interface{}
+	}{
+		{
+			name:     "Set with EXPIREAT command",
+			commands: []string{"SET test_key test_value", "EXPIREAT test_key " + strconv.FormatInt(expireInSeconds, 10)},
```

Comment:
Please use smaller expire timings so that the tests run faster.
Use expire times around 1-2 seconds.